examples/topological-sort/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+var plugin = require('../../index.js');
+var webpack = require('webpack');
+var _ = require('lodash');
+var toposort = require('toposort');
+
+function generate(seed, files) {
+  var nodeMap = {};
+
+  files.forEach(function (file) {
+    nodeMap[file.chunk.id] = file;
+  });
+
+  var edges = [];
+
+  files.forEach(function (file) {
+    if (file.chunk.parents) {
+      file.chunk.parents.forEach(function (parent) {
+        var parentFile = _.isObject(parent) ? nodeMap[parent.id] : nodeMap[parent];
+        if (parentFile) {
+          edges.push([parentFile, file]);
+        }
+      });
+    }
+  });
+
+  const sortedFiles = toposort.array(files, edges);
+
+  return sortedFiles.reduce(function(manifest, file) {
+    return manifest.concat({ name: file.name, path: file.path });
+  }, seed);
+}
+
+
+module.exports = {
+  context: __dirname,
+  entry: {
+    main: './main.js',
+    vendor: './util.js'
+  },
+  output: {
+    filename: '[name].out.js'
```

Comment:
can you set `output.path` to `path.join(__dirname, 'build')` and `filename` to `[name].js`

I've got plan for the future to use those in tests, that will make it easier